### ISSUE TITLE
hhvm 3.14 error

### ISSUE THREAD

**Issue by caaol** [facebook/hhvm#7150]
/usr/bin/hhvm: error while loading shared libraries: libdouble-conversion.so.1: cannot open shared object file: No such file or directory
Processing triggers for libc-bin (2.19-0ubuntu6.5) ...

**Comment by machek** [comment]
Same problem after upgrading to HHVM 3.14.0~trusty on Ubuntu 14.04.4 LTS  AWS EC2

```
/usr/bin/hhvm: error while loading shared libraries: libdouble-conversion.so.1: cannot open shared object file: No such file or directory
```

**Comment by cwultia** [comment]
i think only me who have problem like this..
```
/usr/bin/hhvm: error while loading shared libraries: libdouble-conversion.so.1: cannot open shared object file: No such file or directory
Setting up libpaper-utils (1.1.24+nmu2ubuntu3) ...
Processing triggers for libc-bin (2.19-0ubuntu6.7) ...
Processing triggers for ureadahead (0.100.0-16) ...
```
HHVM 3.14.0~trusty on Ubuntu 14.04.4 LTS DigitalOcean

**Comment by jkluk** [comment]
Yep, same problem.

`/usr/bin/hhvm: error while loading shared libraries: libdouble-conversion.so.1: cannot open shared object file: No such file or directory
Setting up libpaper-utils (1.1.24+nmu2ubuntu3) ...
Processing triggers for libc-bin (2.19-0ubuntu6.9) ...
Processing triggers for ureadahead (0.100.0-16) ...
`

HHVM 3.14.0~trusty on Ubuntu 14.04.4 LTS Azure

**Comment by jkluk** [comment]
Installing liblz4-1 and libdouble-conversion1 appears to resolve the issue on Ubuntu 14.04:

`sudo apt-get install liblz4-1 libdouble-conversion1`

**Comment by jay129** [comment]
oh god same error here. HHVM isn't running nor listening to 9000 nor talk to Nginx anymore.
for 2 hours i thought something i did wrong.

**Comment by javuto** [comment]
Same problem here, and it is being reported through the [fbctf](https://github.com/facebook/fbctf) platform. Confirmed that running ```sudo apt-get install liblz4-1 libdouble-conversion1``` solves the issue.

**Comment by Orvid** [comment]
TLDR; Blame me for the breakage, and then run (as mentioned above):
```
sudo apt-get install liblz4-1 libdouble-conversion1
```


Ah... Whoops... That would be my fault. I had assumed that we already had the dependencies on the packages set to be against the distro packages for double-conversion and lz4, as we should very definitely be using those rather than what we package in third-party, so that any security fixes to them don't have to wait on us to rebuild the HHVM packages, however that doesn't appear to be the case. Distros with a new enough version of libzip in the package manager will also need to install that library as well (14.04 does not, so is using the version we have in third-party).

We already were going to need to ship a 3.14.1 for Debian 8, because apparently GCC version 4.8.2.10 is not the same GCC 4.8.2.10 as is on Ubuntu 14.04 (go figure -_-), which means that the 3.14.0 tag will compile on Ubuntu 14.04 but will not on Debian 8.

The good news is that both of these are simple fixes and the packages for 3.14.1 should all be out by the end of the day tomorrow.

Note to self: Stop breaking things. Oh, and installing packages is bad for your health. Especially ones that HHVM depends on, those are especially bad.

**Comment by soundslocke** [comment]
I ran into this same problem, though slightly different libraries were "missing"...

```hhvm: error while loading shared libraries: libzip.so.4: cannot open shared object file: No such file or directory```

Fixed it by running/installing:

```sudo apt-get install libzip4```

Which then led to:

```hhvm: error while loading shared libraries: libre2.so.1v5: cannot open shared object file: No such file or directory```

Which was fixed by running/installing:

```sudo apt-get install libre2-1v5```

It worked again after that.

**Comment by Orvid** [comment]
Alright, I've got the packaging scripts updated, I'm just waiting to make sure 3.14.1 builds correctly on Debian 8. Once that finishes I'll push out a nightly for Xenial and 14.04 to make sure I haven't broken anything in the packaging scripts before packaging 3.14.1.

**Comment by soundslocke** [comment]
The 3.14.1 release seems good for me.

**Comment by jay129** [comment]
everything works like a charm now. kudos to Orvid and team.

**Comment by javuto** [comment]
Confirmed here as well. All good! Thank you guys!

**Comment by Orvid** [comment]
The last of the debug packages for 3.14.1 are now out.

**Comment by natostanco** [comment]
@orvid xenial is lacking `libnuma1` dep

**Comment by Orvid** [comment]
@natostanco I'll address that as part of fixing https://github.com/hhvm/packaging/issues/144.
